index.js

```diff
@@ -35,14 +33,22 @@ function measuredComponentDidUpdate(nextProps) {
   }
 }
 
+function jsonEqual(a,b) {
+  return JSON.stringify(a) === JSON.stringify(b);
+}
+
 function measureComponents(React) {
   React.Component.prototype.componentDidUpdate = measuredComponentDidUpdate;
 }
 
 function prettyPrintWasted() {
-  const offenders = cloneDeep(
-    pickBy(wastedRenders, component => component.wastedRenderCount > 10)
-  );
+  const offenders = JSON.parse(JSON.stringify(
+    wastedRenders.forEach(component => {
```

Comment:
Lodash's [pickBy](https://lodash.com/docs/4.17.10#pickBy) is closer to `filter` than to `forEach`.  Have you tested this code?